README.md

```diff
@@ -277,6 +278,78 @@ expect(getByTestId(container, 'aria-invalid-false')).not.toBeInvalid()
 
 <hr />
 
+### `toBeLabelled`
+
+```typescript
+toBeLabelled()
+```
+
+This allows you to check if a HTML element is correctly labelled.
+Labelling element is important for accessibility reason, [especially for non-text content](https://www.w3.org/TR/WCAG20/#text-equiv).
+
+An element is labelled if it is having at least one of the following:
+
+- the element is an image having a [not empty `alt` attribute](https://www.w3.org/TR/WCAG20-TECHS/H37.html);
+- the element is a SVG element having [a not empty `title` element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title);
+- the element is having [an `aria-label` attribute](https://www.w3.org/TR/WCAG20-TECHS/ARIA6.html);
+- the element is [referencing a labelling element via `aria-labelledby`](https://www.w3.org/TR/WCAG20-TECHS/ARIA10.html);
+- the element is [a form input having a `title` attribute](https://www.w3.org/TR/WCAG20-TECHS/H65.html);
+- the element is [an image associated to a short content](https://www.w3.org/TR/WCAG20-TECHS/H2.html) within a button or a link;
+- the element is a link, a button or [an object having a text content](https://www.w3.org/TR/WCAG20-TECHS/H53.html).
+
+#### Examples
+
+```html
+<img data-testid="img-alt" src="" alt="Test alt" />
+<img data-testid="img-label" src="" alt="" aria-label="Test alt" />
+<img data-testid="img-labelledby" src="" alt="" aria-labelledby="testId" />
+<img data-testid="img-empty-alt" src="" alt="" />
+<svg data-testid="svg-title"><title>Test title</title></svg>
+<button><img data-testid="img-text-sibling" src="" alt="" /><span>Test content</span></button>
+<button data-testid="button-img-alt"><img src="" alt="Test" /></button>
+<object data-testid="object" data="companylogo.gif" type="image/gif"><p>Company Name</p></object>
+<p><img data-testid="img-paragraph" src="" alt="" />Test content</p>
+<button data-testid="svg-button"><svg><title>Test</title></svg></p>
+<div><svg data-testid="svg-without-title"></svg></div>
+<input data-testid="input-title" title="test" />
+```
+
+##### Using document.querySelector
+
+```javascript
+expect(queryByTestId('img-alt')).toBeLabelled()
+expect(queryByTestId('img-label')).toBeLabelled()
+expect(queryByTestId('img-labelledby')).toBeLabelled()
+expect(queryByTestId('img-empty-alt')).not.toBeLabelled()
+expect(queryByTestId('svg-title')).toBeLabelled()
+expect(queryByTestId('img-text-sibling')).toBeLabelled()
+expect(queryByTestId('button-img-alt')).toBeLabelled()
+expect(queryByTestId('object')).toBeLabelled()
+expect(queryByTestId('img-paragraph')).not.toBeLabelled()
+expect(queryByTestId('svg-button')).toBeLabelled()
+expect(queryByTestId('svg-without-title')).not.toBeLabelled()
+expect(queryByTestId('input-title')).toBeLabelled()
```

Comment:
This block of examples is not implemented with `document.querySelector` and it should.

To be frank, I'm becoming not-a-fan in our README having to maintain this duality of examples for each custom matcher. It makes the README much more lengthy and a bit hard to read. So what if you just keep a single set of examples (the ones using `dom-testing-library`) and that's it. I'll create a separate PR later where I cleanup the rest of the README.

Reply:
Sure 👍 Will do 🙂

Reply:
This remark will become obsolete once #188 is merged. It probably will only be about adopting the same new convention in the README (document with one set of examples only).